stanford-earth/matson
# EARTH-1303: hover state on postcard

# READY FOR REVIEW

# Summary
- Adding a hover state to arrows for postcard

# Needed By (Date)
-Next push

# Urgency
- Non-urgent

# Steps to Test

1. Checkout this branch
2. Navigate to a page that has a postcard (for example, /farm)
3.  Check for color and hover motion change

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- Earth-1303


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
